src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/VisitSchedulerExceptionHandler.kt

```diff
@@ -25,6 +58,28 @@ class VisitSchedulerExceptionHandler {
       )
   }
 
+  @ExceptionHandler(MissingServletRequestParameterException::class)
+  fun handleValidationException(e: MissingServletRequestParameterException): ResponseEntity<ErrorResponse> {
+    log.debug("Bad Request (400) returned", e)
```

Comment:
maybe do e.message? depends I suppose if you want an stack trace in AppInsights in this scenario?